### PR TITLE
Percent Share Sanity Check Fix

### PR DESCRIPTION
**Pull request by eriwarr**
## Description

> This pull request resolves the issues missed in pull request #1943.

**Sanity Check Fixes:**

- Included rows on a data frame whose demographic value equals `"Other."` These rows were previously omitted. 
- Broadened the expected range when evaluating the percent shares.
- Check will not provide the name of the table that is failing. 
- The check will wait until all tables have been reviewed before failing. Previously, the check would fail at the first error.

## Motivation and Context

The previous sanity check needed refactoring as it omitted the necessary rows to run the check.  

## How has this been tested?

The check has been tested with actual CDC data. Below are screenshots of the expected passing and failing tests. 

passing_test

> Failing data should look like this.
failing_test

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality or updates content)

### CODE REVIEW

**Review by joshzarrabi — APPROVED** [review]
Thank you for this eric

File: airflow/dags/util.py [hunk 1]
```diff
@@ -118,17 +118,28 @@ def service_request(url: str, data: dict, **kwargs):
...
 
         query_string = f'SELECT * FROM `{table_name}`'
 
         df: pd.DataFrame = bq_client.query(
             query_string).result().to_dataframe()
-        output = check_pct_values(df)
-        return output
+
+        output = check_pct_values(df, table_name)
+        if output[0] is False:
```
Comment (joshzarrabi) [inline airflow/dags/util.py:134]: This can just be `if not output[0]`

File: airflow/dags/sanity_check.py [hunk 2]
```diff
@@ -34,25 +34,30 @@ def generate_cols(df: pd.DataFrame):
...
 
 
-def check_pct_values(df):
+def check_pct_values(df, table_name):
     # determine cols needed for DF
     std_cols, share_cols, dem_col, df = generate_cols(df)
     cols = std_cols + [dem_col] + share_cols
     df = df[cols]
 
     # remove rows with 'All', 'Unknown', & 'Other as values
```
Comment (benhammondmusic) [inline airflow/dags/sanity_check.py:43]: Update this comment explaining pct share of known

**Review by benhammondmusic — APPROVED** [review]
Sweet

**Pull request merged**